manifests/piped/templates/rbac.yaml

```diff
@@ -33,13 +37,33 @@ metadata:
   labels:
     {{- include "piped.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - '*'
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- nonResourceURLs:
-  - '*'
-  verbs:
-  - '*'
+  {{- include "piped.rbacRules" . | nindent 2 }}
+{{- else if eq .Values.rbac.scope "namespace" -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "piped.fullname" . }}
```

Comment:
Let's specify the namespace for this RoleBinding as well.

Reply:
Oops. Appreciate it.